lib/template/config/initializers/rollbar.rb

```diff
@@ -1,6 +1,7 @@
 Rollbar.configure do |config|
   config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
   config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
+  config.environment = ENV['ROLLBAR_ENVIRONMENT']
```

Comment:
I intentionally left this out, as `ROLLBAR_ACCESS_TOKEN` isn't specified there. Which causes it to default to being disabled.